201-front-door-create-caching/azuredeploy.json

```diff
@@ -60,14 +60,17 @@
                                 "/*",
                                 "/site1/*"
                             ],
-                            "forwardingProtocol": "MatchRequest",
-                            "backendPool": {
-                                "id": "[concat(variables('frontdoorref'),'/backendPools/',parameters('backendPools1').name)]"
-                            },
-                            "CacheConfiguration": {
-                                "queryParameterStripDirective": "StripNone",
-                                "dynamicCompression": "[parameters('dynamicCompression')]"
-                            },
+                            "routeConfiguration": {
+                                "@odata.type": "#Microsoft.Azure.FrontDoor.Models.FrontdoorForwardingConfiguration",
+                                "forwardingProtocol": "MatchRequest",
+                                "backendPool": {
+                                    "id": "[concat(variables('frontdoorref'),'/backendPools/',parameters('backendPools1').name)]"
```

Comment:
remove concat and use resourceId

Reply:
Done, I've updated the PR to use resourceId instead of concat:

"backendPool": {
                                    "id": "[resourceId('Microsoft.Network/frontDoors/backendPools', parameters('frontDoorName'), parameters('backendPools1').name)]"
                                },